src/mode-object.js

```diff
@@ -29,7 +29,6 @@ export function getModeObject(opt_win) {
     esm: IS_ESM,
     test: getMode(opt_win).test,
     log: getMode(opt_win).log,
-    version: getMode(opt_win).version,
```

Comment:
Is `mode-object` use anywhere? We might need to update this to use the core method, so that the object stays the same.

Reply:
Yes, one place (`src/iframe-attributes.js`) in the process of updating and fixing the other test fail